__tests__/app.test.js

```diff
@@ -275,3 +275,54 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("202: updates the current vote by incrementing the value provided in the request", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -179 })
+      .expect(202)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          article_id: 1,
+          votes: -79,
+        });
+      });
+  });
+  test("400: responds with a bad request error if inc_votes property is missing from request", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({  })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  })
+  test("400: responds with a bad request error if inc_votes value is not a number", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: "notNumber" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  })
+  test("400: responds with a bad request error if article_id is not a number", () => {
+    return request(app)
+      .patch("/api/articles/notNumber")
+      .send({ inc_votes: 179 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  })
+  test("400: responds with a not found error if article_id is valid but does not exist", () => {
```

Comment:
Just be careful with your test name, this is a 404 as per your expect statement below 🙂